src/components/auth/profile/CertificatesTab.tsx

```diff
@@ -1,11 +1,155 @@
 import Tab from './Tab'
+import { Box, Checkbox, Button } from '@mui/material'
+import { makeStyles } from '@mui/styles'
+import AdapterDateFns from '@mui/lab/AdapterDateFns'
+import LocalizationProvider from '@mui/lab/LocalizationProvider'
+import DesktopDatePicker from '@mui/lab/DesktopDatePicker'
+import TextField from '@mui/material/TextField'
+import Table from '@mui/material/Table'
+import TableBody from '@mui/material/TableBody'
+import TableCell from '@mui/material/TableCell'
+import TableContainer from '@mui/material/TableContainer'
+import TableHead from '@mui/material/TableHead'
+import TableRow from '@mui/material/TableRow'
```

Comment:
You can combine the imports to reduce the repetition
```suggestion
import {
  TextField,
  Table,
  TableBody,
  TableCell,
  TableContainer,
  TableHead,
  TableRow,
} from '@mui/material'
```